### PR TITLE
feat(plugins) Hide plugins that also have integrations

### PR DESCRIPTION
**Pull request by markstory**
By hiding legacy plugins in the UI we make it a bit harder for customers to enable plugins we don't want them to use. This should help guide them towards the organization integrations which we plan on supporting more.

I added a new js fixture for the hidden plugin and the existing snapshot tests ensure that the hidden plugin doesn't display by not changing.

Fixes APP-890

### CODE REVIEW

**Comment by billyvg** [comment]
What happens if a user has a hidden plugin that is enabled, will they be allowed to disable it?

**Comment by markstory** [comment]
@billyvg Yes it will be visible and can be disabled. Once disabled it is gone forever though 😬

File: tests/sentry/api/endpoints/test_project_plugins.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                'project_slug': project.slug,
+            }
+        )
+        response = self.client.get(url)
+        assert response.status_code == 200, (response.status_code, response.content)
+        assert len(response.data) >= 9
+
+        auto_tag = response.data[0]
+        assert auto_tag['name'] == 'Auto Tag: Browsers'
+        assert auto_tag['enabled'] is True
```
Comment (lauryndbrown) [inline tests/sentry/api/endpoints/test_project_plugins.py]: So, I think it's important to have a test where the plugin is hidden. A front end test as well would be useful.

File: src/sentry/plugins/__init__.py [hunk 2]
```diff
@@ -7,6 +7,14 @@
 """
 from __future__ import absolute_import
 
+HIDDEN_PLUGINS = (
+    'bitbucket',
+    'gitlab',
+    'github',
+    'slack',
+    'vsts'
```
[the remark this answers is not in the document]
Comment (lauryndbrown) [reply src/sentry/plugins/__init__.py]: Woops I see it's the slug. never mind then :)

File: src/sentry/plugins/__init__.py [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
+)
```
[the remark this answers is not in the document]
Comment (markstory) [reply src/sentry/plugins/__init__.py]: Good idea. I put a task in Jira as those are easier to remember/find than code comments (APP-895).

**Review by lauryndbrown — APPROVED** [review]

File: tests/sentry/api/endpoints/test_project_plugins.py (hunk 1 above)
Comment (markstory) [reply tests/sentry/api/endpoints/test_project_plugins.py]: Test added.

**Pull request merged**